src/System.Runtime.Caching/tests/AdditionalCacheTests/AdditionalCacheTests.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+using System;
+using System.Runtime.Caching;
+
+namespace System.Runtime.Caching.Tests
+{
+    // These are the tests to fill in some of the coverage in ported Mono caching tests
+    public class AdditionalCacheTests
+    {
+        [Fact]
+        public void DisposedCacheTest()
+        {
+            var mc = new MemoryCache("my disposed cache 1");
+            mc.Add("aa", "bb", new CacheItemPolicy());
+            mc.Dispose();
+
+            Assert.Null(mc["aa"]);
+
```

Comment:
Does MemoryCache simply return null if used after it's been disposed instead of throwing ObjectDisposedException?

Reply:
Returns null. This was a design decision made for the desktop System.Runtime.Caching long time ago and we're keeping the same behavior for compatibility reasons.